### PR TITLE
initial tex template; has some issues in appendix section

### PR DESCRIPTION
**Pull request by DiptoDas8**
This template works. There are only two known issues with this template. I have revised the appendix section accordingly that is not reflected in this version. This template does not have non-English language support, on which I am working currently.

### CODE REVIEW

File: missouri_state_thesis_main.aux [hunk 1]
```diff
@@ -0,0 +1,45 @@
+\relax
```
Comment (anthonyjclark) [inline missouri_state_thesis_main.aux]: The generated files should not be pushed to this repository. It would be good to add them to the `.gitignore` file.

See here: https://www.gitignore.io/api/latex

File: missouri_state_thesis_main.tex.backup [hunk 2]
```diff
@@ -0,0 +1,63 @@
+\documentclass[12pt,notitlepage,oneside]{report}
```
Comment (anthonyjclark) [inline missouri_state_thesis_main.tex.backup]: Is this file needed in the repo?

File: missouri_state_thesis_old.sty [hunk 3]
```diff
@@ -0,0 +1,286 @@
+% You should not change anything in this file unless you really know
```
Comment (anthonyjclark) [inline missouri_state_thesis_old.sty]: Old style?

**Review by anthonyjclark — CHANGES_REQUESTED** [review]
Looks great Dipto. Thanks! Just need to remove a few files.

File: missouri_state_thesis_main.tex.backup (hunk 2 above)
Comment (DiptoDas8) [reply missouri_state_thesis_main.tex.backup]: no, I removed this file.

File: missouri_state_thesis_old.sty (hunk 3 above)
Comment (DiptoDas8) [reply missouri_state_thesis_old.sty]: yes, I removed this file as well.

**Review by anthonyjclark — APPROVED** [review]
